definitions/divorce/json/CaseField/CaseField-nonprod.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "LiveFrom": "23/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "HelpWithFeesReferenceNumber",
+    "Label": "Help with fees reference",
+    "FieldType": "Text",
+    "RegularExpression": "([Hh][Ww][Ff]-?)?[0-9a-zA-Z]{3}-?[0-9a-zA-Z]{3}$",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "23/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "FeeAccountReferenceNumber",
+    "Label": "Enter your account number",
+    "HintText": "Example: PBA0896366",
+    "FieldType": "Text",
+    "RegularExpression": "^([Pp][Bb][Aa])\\d{7}$",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "23/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "FurtherFeeAccountReferenceNumbers",
+    "Label": "Further Fee account numbers",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "ReferenceNumber",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "23/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "FurtherHelpWithFeesReferenceNumbers",
+    "Label": "Further HWF references",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "ReferenceNumber",
+    "SecurityClassification": "Public"
+  }
+]
```

Comment:
Definitions for new reference fields and collection types for both HWF and Fee Accounts